techniques/html/H30.html

```diff
@@ -13,50 +13,50 @@ <h1>Providing link text that describes the purpose of a link for anchor elements
 		</section>
 		<section id="applicability">
 			<h2>When to Use</h2>
-      <p><abbr title="HyperText Markup Language">HTML</abbr> documents that contain links, (<code class="language-html">&lt;a href&gt; elements)</code></p>
+      <p><abbr title="HyperText Markup Language">HTML</abbr> documents that contain links, (<code>&lt;a href&gt; elements)</code></p>
```

Comment:
```suggestion
      <p><abbr title="HyperText Markup Language">HTML</abbr> documents that contain links, (<code>&lt;a href&gt;</code> elements)</p>
```

could benefit from highlighting